spec/models/payment_confirmation_spec.rb

```diff
@@ -43,6 +43,17 @@
 
       expect(payroll_run.reload.confirmation_report_uploaded_by).to eq(admin_user_id)
     end
+
+    it "queues emails" do
+      travel_to Date.parse("2019-01-01") do
```

Comment:
Maybe we should name this date, e.g. `a_tuesday`, and then rename the next one to `the_following_friday` to make the intent clearer?